packages/nextjs/hooks/scaffold-eth/useDeployedContractInfo.ts

```diff
@@ -14,37 +14,42 @@ type GeneratedContractType = {
 export const useDeployedContractInfo = (contractName: string | undefined | null) => {
   const configuredChain = getTargetNetwork();
   const [deployedContractData, setDeployedContractData] = useState<undefined | GeneratedContractType>(undefined);
+  const [loading, setLoading] = useState(true);
```

Comment:
Setted initial state as `true` because setting initial state as `false` again shows flash of `No contract founde!` on the UI